styles/container.scss

```
@@ -0,0 +1,27 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: auto;
+  min-width: 340px;
+  padding: 30px;
+  width: min-content;
+  box-shadow: 0 0 6px rgba(0, 0, 0, 0.14);
```

HEX тоже позволяет задействовать альфа канал - можно использовать для единообразия и вынести в переменные
```suggestion
  box-shadow: 0 0 6px #00000014;
```